docs/en_US/Tutorial/Installation.md

```diff
@@ -1,6 +1,18 @@
-# Installation of NNI
+# Plan your topology
```

Comment:
The topology concept makes end users feeling complex, and NNI is not so complex actually. I suggest not to add this section. If people want to implement new trainning platform, they need to know this. So it's more like design related things, shouldn't be put in installation part.

Reply:
agree with @squirrelsc

Reply:
@squirrelsc  @QuanluZhang - agree the concern. while still think we need to have this document in a lower exposure place. If you agree this, where would you like to recommend for putting it?